src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -5,16 +5,23 @@ import { toTitleCase } from "@artsy/to-title-case"
 import artworkMediums from "lib/artworkMediums"
 import allAttributionClasses from "lib/attributionClasses"
 import { COLORS } from "lib/colors"
+import { round } from "lodash"
 
-// Taken from Force's SizeFilter component
-export const SIZES = {
+export const SIZES_IN_CM = {
   SMALL: "Small (under 40cm)",
   MEDIUM: "Medium (40 – 100cm)",
   LARGE: "Large (over 100cm)",
 }
+export const SIZES_IN_INCHES = {
+  SMALL: "Small (under 16in)",
+  MEDIUM: "Medium (16in – 40in)",
+  LARGE: "Large (over 40in)",
+}
 
 const ONE_IN_TO_CM = 2.54
 
+const DEFAULT_METRIC = "in"
```

Comment:
**question (non-blocking):** Do we think this duplicates the default declared here? 

https://github.com/artsy/metaphysics/blob/9c9bcbb12bacfe3fabc83e9c249d034f2f4af31a/src/schema/v2/me/index.ts#L94

If this is truly the same concept, probably better to declare it once and import where necessary.

Reply:
Good idea!